CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ For older changes see the [archived Singularity change log](https://github.com/a
 - Added ability to set a custom config directory via the new
   `APPTAINER_CONFIGDIR` environment variable.
 
+### Other changes
+
+- Able to change log level through environment variables.
+
```

Comment:
Put the changelog entry under `Changes since last pre-release` since we'll be including this in 1.2.0-rc.2.  

Change "Able" to "Added ability" and append "APPTAINER_SILENT", "APPTAINER_QUIET", and "APPTAINER_VERBOSE".

Reply:
got it!
reflected here
https://github.com/apptainer/apptainer/pull/1472/commits/157b946964534ab2b1cb139b399bf3b4fc8893b5